app/models/grda_warehouse/tasks/chronically_homeless.rb

```diff
@@ -52,6 +52,48 @@ def initialize(
       @debug = debug
     end
 
+    def chronic_on_date(client_id)
+      debug_log "Calculating chronicity for #{client_id}"
+      # All chronically homeless must also be disabled
+      return false unless disabled?(client_id)
+
+      # remove any cached calculations from the previous client
+      reset_for_batch
+      adjusted_homeless_dates_served = residential_history_for_client(client_id: client_id)
+      homeless_months = adjusted_months_served(dates: adjusted_homeless_dates_served)
+      homeless_months_size = if homeless_months.size > 36 then 36 else homeless_months.size end
```

Comment:
Do we want to use this logic of capping `months_homeless` at 36? Previously we had the logic but didn't use it.

Reply:
Not sure it matters. You can't have more than 36 months in the past 3 years, which is all the chronic calculation would need.

Reply:
Ok. Removed it since it wasnt used before